### PR TITLE
Fix/eslint stricter

### PR DESCRIPTION
**Pull request by mattvb91**
small cleanup to make eslint stricter

### CODE REVIEW

File: packages/base/src/Bootstrap.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function isAPlugin(packageName?: string) {
     }
 
     const packageJson = require(packageName + "/package.json") as Object;
-    if (!packageJson.hasOwnProperty("corejam")) return false;
+    if (!Object.prototype.hasOwnProperty.call(packageJson, "corejam")) return false;
```
Comment (patrickhaug) [inline packages/base/src/Bootstrap.ts:27]: Which lint rule? That is ugly as hell. Could you use "key in obj"?

**Review by patrickhaug — CHANGES_REQUESTED** [review]
Do you want to make use of object prototype calls?

File: packages/base/src/Bootstrap.ts (hunk 1 above)
Comment (mattvb91) [reply packages/base/src/Bootstrap.ts:27]: https://eslint.org/docs/rules/no-prototype-builtins

Comment (mattvb91) [reply packages/base/src/Bootstrap.ts:27]: and yea agreed that it looks ugly as hell. Im happy to just keep the rule off

**Review by patrickhaug — APPROVED** [review]

**Pull request merged**